event-consumer/manifest.yml

```diff
@@ -0,0 +1,35 @@
+packages:
+  __APP_PACKAGE__:
+    license: Apache-2.0
+    actions:
+      generic:
```

Comment:
maybe rename this action to a more meaningful name than `generic`?

Reply:
rename it to event-consumer